packages/serialization/json/src/jsonSerializationWriter.ts

```diff
@@ -118,7 +118,7 @@ export class JsonSerializationWriter implements SerializationWriter {
 
     this.onStartObjectSerialization &&
       this.onStartObjectSerialization(value as unknown as Parsable, this);
-    value && serializerMethod && serializerMethod(this, value);
+    value && serializerMethod(this, value);
```

Comment:
no sure about this change, we want to test whether the method reference has been passed before calling it.

Reply:
I agree, I have reverted the change